language/move-prover/bytecode/src/access_path.rs

```diff
@@ -469,10 +472,20 @@ impl Offset {
                 // we conflate address and signer, so this can happen
                 s.get_type()
             }
-            _ => panic!(
-                "Invalid base type {:?} for offset {:?} in get_type",
-                base, self
-            ),
+            (Type::Error, _) => {
+                // couldn't infer the type of `base`. propagate the error
+                Type::Error
```

Comment:
This change is to allow display printing of local access paths, which helps a lot with debugging.